libinstpatch/IpatchConverter.c

```diff
@@ -68,7 +69,29 @@ G_LOCK_DEFINE_STATIC(conv_maps);
 static GList *conv_maps = NULL;	/* list of all registered IpatchConverterInfo */
 static gpointer parent_class = NULL;
 
+/*------ Initialization/deinitialization of converter system ----------------*/
+/* Initialize converter system (conv_maps static list) */
+void _ipatch_converter_init(void)
+{
+    /* list of all registered IpatchConverterInfo */
+    conv_maps = NULL;
+}
+
+/* Free converter system */
+void _ipatch_converter_deinit(void)
+{
+    /* free list of all registered IpatchConverterInfo */
+    g_list_free_full(conv_maps,
```

Comment:
`g_list_free_full` was introduced in glib 2.28. This breaks the CI build on Win64.

Reply:
Have a look at deed052c82e854e7086662b14cc82f6fd7d00aff.

Reply:
Ok, thanks. I will  replace g_list_free_full().